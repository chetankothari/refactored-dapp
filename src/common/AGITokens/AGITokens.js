import React from 'react';
import PropTypes from 'prop-types';

const AGITokens = ({ children }) => {
  return (
    <span>
      {children} AGI
    </span>
  );
};

AGITokens.propTypes = {
  children: PropTypes.string.isRequired
};

export default AGITokens;
