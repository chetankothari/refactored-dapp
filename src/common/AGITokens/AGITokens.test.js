import React from 'react';
import { shallow } from 'enzyme';

import AGITokens from "./AGITokens";

describe('AGITokens', function() {
  it('renders as expected', () => {
    const wrapper = shallow(<AGITokens>0.00000001</AGITokens>);

    expect(wrapper).toMatchSnapshot();
  });
});
