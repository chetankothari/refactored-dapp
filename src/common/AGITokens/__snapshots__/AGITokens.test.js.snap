// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AGITokens renders as expected 1`] = `
<span>
  0.00000001
   AGI
</span>
`;
